examples/java/src/test/java/dev/selenium/drivers/OptionsTest.java

```diff
@@ -47,5 +47,18 @@ public void setPageLoadStrategyNone() {
       driver.quit();
     }
   }
+
+  @Test
+  public void setAcceptInsecureCerts() {
+    ChromeOptions chromeOptions = new ChromeOptions();
+    options.setAcceptInsecureCerts(true);
```

Comment:
chromeOptions.setAcceptInsecureCerts(true);

Reply:
My bad. I have updated it to the right reference chromeOptions